var/spack/repos/builtin/packages/bash/package.py

```diff
@@ -16,16 +16,31 @@ class Bash(AutotoolsPackage, GNUMirrorPackage):
 
     maintainers = ["adamjstewart"]
 
+    version("5.2", sha256="a139c166df7ff4471c5e0733051642ee5556c1cc8a4a78f145583c5c81ab32fb")
     version("5.1", sha256="cc012bc860406dcf42f64431bcd3d2fa7560c02915a601aba9cd597a39329baa")
     version("5.0", sha256="b4a80f2ac66170b2913efbfb9f2594f1f76c7b1afd11f799e22035d63077fb4d")
     version("4.4", sha256="d86b3392c1202e8ff5a423b302e6284db7f8f435ea9f39b5b1b20fd3ac36dfcb")
     version("4.3", sha256="afc687a28e0e24dc21b988fa159ff9dbcf6b7caa92ade8645cc6d5605cd024d4")
 
     depends_on("ncurses")
+    depends_on("readline@8.2:", when="@5.2:")
     depends_on("readline@5.0:")
     depends_on("iconv")
+    depends_on("gettext")
```

Comment:
Previous releases built without this, but the latest release fails:
```
/var/folders/j1/68dlgpr91vlgs26vty2c8xk80000gn/T/ajstewart/spack-stage/spack-stage-bash-5.2-y2ki4v2b74dswufi3cdihigzbsnwoedh/spack-src/include/gettext.h:27:11: fatal error: 'libintl.h' file not found
# include <libintl.h>
          ^~~~~~~~~~~
```
Adding this dep solves the issue in 5.2. In 5.1 and older, bash does link to libintl, so I think we should keep the dependency for all versions.